### PR TITLE
fix regex name and age

### PR DESCRIPTION
**Pull request by gutoxp**
Changed the regex pattern. I believe it works. Unless some Overwatch trainee changes another hero bio again :sweat_smile:

### CODE REVIEW

**Comment by gutoxp** [comment]
fix #9

File: overwatch/parser.py [hunk 1]
```diff
@@ -43,7 +43,8 @@ def extract_url_params(self, response):
 
     def extract_bio(self, response):
         content = response.find("li", class_="name").span
-        realname, age = re.search(GET_REAL_NAME_AND_AGE, content.text).groups()
+        _, realname, _, age = re.search(GET_REAL_NAME_AND_AGE, content.text).groups()
```
Comment (marcuxyz) [inline overwatch/parser.py]: You can use .group() function for eliminate the `_` variables. Passing this params: 2, 4. e.g:

`realname, age = re.search(GET_REAL_NAME_AND_AGE, content.text).group(2,4)`

**Review by marcuxyz — CHANGES_REQUESTED** [review]
Modify the code for make it clean

File: overwatch/parser.py (hunk 1 above)
Comment (gutoxp) [reply overwatch/parser.py]: Cool ! i will do that

**Pull request merged**